src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6783,8 +6783,14 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExpressionTreeContainsCollectionLiteral" xml:space="preserve">
     <value>An expression tree may not contain a collection literal.</value>
   </data>
-  <data name="ERR_CollectionLiteralElementNotImplemented" xml:space="preserve">
-    <value>Support for collection literal dictionary and spread elements has not been implemented.</value>
+  <data name="ERR_CollectionLiteralKeyValueOrSpreadExpected" xml:space="preserve">
+    <value>A dictionary collection literal can contain key-value pairs and spread elements only.</value>
+  </data>
+  <data name="ERR_CollectionLiteralDictionarySpreadUnexpectedType" xml:space="preserve">
+    <value>A dictionary collection literal spread must be a collection of 'System.Collections.Generic.KeyValuePair&lt;,&gt;'.</value>
```

Comment:
Not true :)  if the target dictionary has an `Add(element_type)` that was expected to work.